app/views/unattended/provisioning_templates/discovery/debian_kexec.erb

```diff
@@ -34,7 +34,7 @@ require:
   options << 'console-setup/ask_detect=false console-setup/layout=USA console-setup/variant=USA keyboard-configuration/layoutcode=us localechooser/translation/warn-light=true localechooser/translation/warn-severe=true'
   options << "locale=#{host_param('lang') || 'en_US'}"
   extra = []
-  extra << "--kexec-file-syscall" if @host.pxe_loader.include?('SecureBoot')
+  extra << "--kexec-file-syscall" if (@host.pxe_loader | '').include?('SecureBoot')
```

Comment:
Looking a the other changes I think this was a typo.
```suggestion
  extra << "--kexec-file-syscall" if (@host.pxe_loader || '').include?('SecureBoot')
```

Reply:
Also, we can use the safe operator now so this also works in develop (but not 3.9 or older):
```suggestion
  extra << "--kexec-file-syscall" if @host.pxe_loader&.include?('SecureBoot')
```